### ISSUE TITLE
Empty page on error

### ISSUE THREAD

**Issue by sebastiaanluca** [GrahamCampbell/Laravel-Exceptions#34]
So I installed the package following the instructions (like I did many times before). Though with this project I get an empty page and an 500 internal server error when it encounters an exception.

Error (after the not found or any other exception): `exception 'ReflectionException' with message 'Class  does not exist'`. Traced it back to `\GrahamCampbell\Exceptions\ExceptionHandlerTrait::getDisplayer` where `$this->config` is `null` and it's trying to create a class `null` using `$this->make()` on line 121.

Now I know that's not supposed to be, so I traced back all my steps and tried from scratch a few times. No dice. Anything I'm missing? Laravel 5.1, `"graham-campbell/exceptions": "~6.0"`, extended your handler, etc.

---

`$container->config` in `\GrahamCampbell\Exceptions\ExceptionHandler` on line 46 has no `exceptions` key.

**Comment by GrahamCampbell** [comment]
Have you tried deleting the cached files from storage, including the services.json file?

**Comment by sebastiaanluca** [comment]
Didn't have any cache (using memcached), but switching to file doesn't change anything. Neither does deleting the services.json file.

**Comment by GrahamCampbell** [comment]
> Didn't have any cache (using memcached), but switching to file doesn't change anything. 

That's not what I meant. :) I mean the files laravel creates in the storage folder. Like the compiled php file and the services.json file.

**Comment by GrahamCampbell** [comment]
> Traced it back to \GrahamCampbell\Exceptions\ExceptionHandlerTrait::getDisplayer where $this->config is null

How is that possible? Is your exception thrown before the config is initialized?

**Comment by sebastiaanluca** [comment]
Yeah I know, should've said that more clearly. I've deleted all the cache files (there was only services.json), even ran `php artisan cache:clear` and all the other cache-clearing commands.

I have no idea how that's possible, that's what I'm asking :p I have some home-made modules where routes get mapped in the boot method of their service providers, but I've disabled those without any change. Moved your service provider up the chain in config/app.php, again, without change.

I'm testing this with a simple inexistent route btw. Default Laravel error page works.
